tests/template_backends/test_jinja2.py

```diff
@@ -96,3 +96,39 @@ def test_dirs_pathlib(self):
         })
         template = engine.get_template('hello.html')
         self.assertEqual(template.render({'name': 'Joe'}), 'Hello Joe!')
+
+    def test_template_render_nested_error(self):
+        template = self.engine.get_template('template_backends/syntax_error_include.html')
+        with self.assertRaises(TemplateSyntaxError) as e:
+            template.render(context={})
+        debug = e.exception.template_debug
+        self.assertEqual(debug['after'], '')
+        self.assertEqual(debug['before'], '')
+        self.assertEqual(debug['during'], '{% block %}')
+        self.assertEqual(debug['bottom'], 1)
+        self.assertEqual(debug['top'], 0)
+        self.assertEqual(debug['line'], 1)
+        self.assertEqual(debug['total'], 1)
+        self.assertEqual(len(debug['source_lines']), 1)
+        self.assertTrue(debug['name'].endswith('syntax_error.html'))
+        self.assertIn('message', debug)
```

Comment:
Jinja raises `jinja2.TemplateSyntaxError` in `render()` not in get_template()` when an error is in the included template, so that's the real usage. We don't need to mock anything here.